system/jlib/jptree.cpp

```diff
@@ -2986,7 +2986,7 @@ void LocalPTree::setAttribute(const char *key, const char *val)
     else
     {
         attrs = (AttrValue *)realloc(attrs, (numAttrs+1)*sizeof(AttrValue));
-        v = &attrs[numAttrs++];
+        v = new(&attrs[numAttrs++]) AttrValue;  // Initialize new AttrValue
```

Comment:
I _could_ take this out - it's not really needed as the set wil always set something. But it's cleaner and less error-prone for future